app/screens/auth/CreateWallet.tsx

```diff
@@ -170,7 +170,7 @@ const CreateWallet = ({ history, location }: Props) => {
       </LoaderWrapper>
     );
   }
-  const header = subMode === 1 ? 'SET NEW WALLET PASSWORD' : `${walletName.displayName} CREATED`;
+  const header = subMode === 1 ? 'SET NEW WALLET PASSWORD' : 'MAIN WALLET CREATED';
```

Comment:
don't change to hardcoded wallet name, just add to CorneredContainer css property "text-transform: capitalize;"